examples/mobile/ScheduleApp/js/app.js

```diff
@@ -96,6 +95,8 @@
 
 			dropdownmenuWidget.open();
 		});
+
+		this._buildEventList(page.querySelector(".ui-scrollview-view"));
 		// @todo read data from storage
 		// @todo fill page content
 		// @todo refresh page
```

Comment:
After change of page content developer should called `refresh` method on page... but I don't understand why this doesn't work. Instead of `refresh` method we can use `createWidgets` for build all widget in page content.

```
// @todo refresh page
tau.engine.createWidgets(page);
```